projects/charger/inc/charger_pin.h

```diff
@@ -1,9 +1,12 @@
 #pragma once
-// Module for interfacing with the charger state pin.
-// Requires: event_queue, interrupts, gpio and gpio_it to be initialized
+// Module for interfacing with the charger proximity pin..
+// Requires: event_queue, gpio, soft timers and ADC.
```

Comment:
Should probably explain what this pin is expecting?

Reply:
Done.